Sources/MeiliSearch/Model/Update.swift

```diff
@@ -49,4 +49,43 @@ public struct Update: Codable, Equatable {
 
     }
 
+    public enum Status: Codable, Equatable {
+
+        case enqueued
+        case processed
+        case failed
+
+        public enum CodingError: Error {
```

Comment:
```suggestion
        public enum StatusError: Error {
```
What do you think of this less generic name ?

Reply:
Replaced it with `StatusError.unknown`.